srcpkgs/ly/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'ly'
+pkgname=ly
+version=0.5.2
+revision=1
+_argoat_githash=0eb7afae7a001094c9166a8959e021375707522c
+_testoasterror_githash=ee7c9d031d4632a6f381a6c174a38539bac04068
+_configator_githash=8cec1786196ae6f6a8b35e66181277457f2a2bb2
+_ctypes_githash=eb4b36559de8d17015f9416861ed085d06f5f5ed
+_dragonfail_githash=0a2492c6aab3ff64e182db894ce4d40b26799fbd
+_termbox_githash=23fff64470b0730959b39c70aa31fbddd776d9bc
+build_style=gnu-makefile
+make_build_args="FLAGS=\"$CFLAGS\""
+conf_files="/etc/ly/config.ini"
+hostmakedepends="git"
```

Comment:
Does it still need `git`?

Reply:
No, but adding it shuts up the "git not found" output that pops up without it. Granted, you still get the "fatal, not a git repository" error with it installed, so I guess it doesn't matter. I'll remove it.